assets/css/woocommerce.scss

```
@@ -26,14 +26,14 @@ p.demo_store,
 	font-size: 1em;
 	padding: 1em 0;
 	text-align: center;
-	background-color: $primary;
-	color: $primarytext;
+	background-color: var( --wc-primary );
+	color: var( --wc-primary )text;
```

I think Claudio already flagged, but we need to clear up instances like this in order for Grunt to compile things successfully:

```suggestion
	color: var( --wc-primary );
```

(If we can successfully compile, it will be easier to evaluate this change 😃 )